### PR TITLE
adding Votes model and model test

### PR DESCRIPTION
**Pull request by mcon85**
Added model test for votes, created votes migration, created votes model

### CODE REVIEW

File: app/models/vote.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Vote < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :review
+
+  VOTES = ["up", "down"].freeze
+  validates :vote_type, presence: true, inclusion: { in: VOTES }
```
Comment (jraenorris) [inline app/models/vote.rb]: nice

File: db/migrate/20160708144348_create_votes.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CreateVotes < ActiveRecord::Migration
+  def change
+    create_table :votes do |t|
+      t.belongs_to :user, index: true
```
Comment (jraenorris) [inline db/migrate/20160708144348_create_votes.rb]: Can there be a vote with no user/ review? Or should these be `null: false`

File: app/models/vote.rb (hunk 1 above)
Comment (jraenorris) [inline app/models/vote.rb]: What about validating presence of user and review?

Comment (freyjasdottir) [inline app/models/vote.rb]: Addressed in this commit: https://github.com/mcon85/doors_at_eight/pull/28/commits/76622824d55f469fb3e2a2662354f533eb27bb8a

**Pull request merged**